src/pages/public_registration/DeletedPublicationInformation.tsx

```diff
@@ -1,30 +1,76 @@
 import { Registration, RegistrationStatus } from '../../types/registration.types';
-import { Box, Typography } from '@mui/material';
+import { Box, Link, Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
+import { getTitleString } from '../../utils/registration-helpers';
+import { useQuery } from '@tanstack/react-query';
+import { fetchRegistration } from '../../api/registrationApi';
```

Comment:
Mener @joachimjorgensen også bruker IntelliJ, og fikk på plass automatisk sortering av imports, så går an å høre hvilke triks han har!
Syns det er fint å ha alle eksterne ting øverst, og våre ting nederst. I tillegg slipper man å få en diff bare fordi noen har flyttet på imports feks

Reply:
Jeg spurte Anette om tips til dette 😅

Reply:
Do as I say, not what I do.

Neida, jeg skrudde på settings for å fjerne unused imports, er en settings til å å arrangere de?

Reply:
vscode har i hvert fall det, og bare antar at andre populære IDEs har samme mulighet, men har ikke brukt det andre plasser selv 🤔

Reply:
Jeg har heller ikke funnet en setting som arrangerer imports... Bare rydder i stylingen av de